README.md

```diff
@@ -6,56 +6,27 @@ We use a template repository approach for maintainability.
 
 ## Creating a repository for an individual training workshop
 
-First, you will need to create a new repository.
+First, you will need to create a new repository using the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).
+This will create a new repository with the same layout as this template repository under the given organization or user you choose to administer the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should administer the new repository.
 
-### Data Lab workshops
+We recommend setting the new repository to public.
+(When you use GitHub pages on a private repository, those pages are publicly accessible anyway.)
```

Comment:
So, this is tricky: for outside users without paid plans, the repo _must_ be public to use Pages. I don't know if we want to get into it more than we did before. (But also, capitalize "Pages"?)

Reply:
First I have a typo there - should be "setting the new repo to **private**."

That said, I don't think what you wrote it true?

https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages

> Warning: GitHub Pages sites are publicly available on the internet, even if the repository for the site is private. If you have sensitive data in your site's repository, you may want to remove the data before publishing. For more information, see "[About repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories#about-repository-visibility)."

Reply:
Yes but also (from the green box): 
> GitHub Pages is available in public repositories with GitHub Free and GitHub Free for organizations, and in public and private repositories with GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server.

Reply:
Ah, I see... yeah..

Reply:
I've added more detail here, let me know what you think!